### ISSUE TITLE
Playlist m3u

### ISSUE THREAD

**Issue by Gamefire238** [anilbeesetti/nextplayer#934]
It would be great if the app could also open and run m3u Playlists

**Comment by VasilisKos** [comment]
This is a VIDEO player.

**Comment by ghost** [comment]
Support Live Streaming (m3u) and ( Play online magnet links video) Very Good
### m3u list
https://github.com/iptv-org/iptv

### magnet
You can refer to this torrent library

https://github.com/RangerMauve/bt-fetch
https://github.com/RangerMauve/bt-mut

https://github.com/webtorrent

**Comment by ghost** [comment]
In order to play Playlist m3u, you need to install other player VLC kodi. 
Their players also have problems with format decoding, consume more resources on the phone and have a complicated interface.

**Comment by gift2010** [comment]
> It would be great if the app could also open and run m3u Playlists

agreed, i need this great player to support m3u playlist so i can watch tv live shows. 
like apple store app aptv

**Comment by sam-reza** [comment]
👍 kinda agreed

**Comment by Gamefire238** [comment]
I'll close my request, well it's a nice project, but for some reason I've never actually used it now, well maybe my request was a bit too much, Apologies for any hassle
